server/aws-lsp-partiql/src/server/language-service.ts

```diff
@@ -0,0 +1,39 @@
+import type { Diagnostic } from 'vscode-languageserver-protocol'
```

Comment:
nit: imports from `vscode-languageserver-protocol` should be available from  `@aws/language-server-runtimes/server-interface`